src/main/java/com/powsybl/openloadflow/sensi/DcSensitivityAnalysis.java

```diff
@@ -192,7 +195,94 @@ private DenseMatrix initRhs(LfNetwork lfNetwork, EquationSystem equationSystem,
         return factorsByVarConfig;
     }
 
-    public List<SensitivityValue> analyse(Network network, List<SensitivityFactor> factors, OpenLoadFlowParameters lfParametersExt,
+    /**
+     * If the slack is distributed, we need to create a factor for every generator
+     * @param network
+     * @param lfNetwork
+     * @param oldFactors
+     * @return
+     */
+    private List<SensitivityFactor> getDistributedFactors(Network network, LfNetwork lfNetwork, List<SensitivityFactor> oldFactors) {
```

Comment:
this method takes the network as input, because I need to extract generators. But this may be an issue, so I'd like to use just lfNetwork, but I do not know how to extract generators from it